spec/controllers/reservations_controller_spec.rb

```diff
@@ -6,28 +6,27 @@
   render_views
 
   before(:all) do
-    @app_config = FactoryGirl.create(:app_config)
-
     @user = FactoryGirl.create(:user)
     @banned = FactoryGirl.create(:banned)
     @checkout_person = FactoryGirl.create(:checkout_person)
     @admin = FactoryGirl.create(:admin)
   end
 
   after(:all) do
-    User.delete_all
-    AppConfig.delete_all
-    Reservation.delete_all
-    Category.delete_all
-    EquipmentModel.delete_all
-    EquipmentItem.delete_all
+    User.destroy_all
+    Category.destroy_all
+    EquipmentModel.destroy_all
+    EquipmentItem.destroy_all
   end
 
   before(:each) do
+    @ac = mock_app_config(**FactoryGirl.attributes_for(:app_config),
```

Comment:
Is there a reason why we need to pass in all of these attributes? We're stubbing all the relevant methods later so I'm not sure why we need to specify them now. Or is it the case that we need `AppConfig.first` to be functional for `ability.rb` / since these controller specs aren't fully mocked / stubbed?